ci-automation/packages-tag.sh

```diff
@@ -120,5 +126,6 @@ function _packages_tag_impl() {
       create_versionfile "$sdk_version" "$version"
     )
     update_and_push_version "${version}" "${push_branch}"
+    apply_local_patches
```

Comment:
Isn't applying local patches in `packages-tag.sh` pointless? They will be applied in `packages.sh`, just before the build.

Reply:
Yes, I just wondered whether it still makes sense to have it when someone copies this job to do something else

Reply:
Maybe applying embargoed patches should be a separate step (a separate script in `ci-automation` directory) so that "someone" could just invoke it before doing this "something else". On the other hand, you have already merged the jenkins changes and you also have made applying those patches idempotent when they were already applied (so invoking ci-automation scripts locally won't fail there), so let's avoid the hassle for now.